styles/PingCAP/Gender.yml

```
@@ -0,0 +1,9 @@
+extends: existence
+message: "Don't use '%s' as a gender-neutral pronoun."
```

Would it be necessary to tell users what word to use?
```suggestion
message: "Don't use '%s' as a gender-neutral pronoun. Instead, use the singular 'they'."
```